staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -965,6 +965,22 @@ type PodSecurityPolicySpec struct {
 	// privilege escalation.
 	// +optional
 	AllowPrivilegeEscalation bool `json:"allowPrivilegeEscalation,omitempty" protobuf:"varint,16,opt,name=allowPrivilegeEscalation"`
+	// AllowedHostPaths is a white list of allowed host paths. Empty indicates that all host paths may be used.
```

Comment:
For an example of a type documented correctly, see ScaleStatus.

It doesn't need to be fixed for this PR, but at some point we're going to have to block all further API changes until someone addresses it.